pkgs/tools/text/reckon/default.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv {
+  name = "reckon";
+  inherit ruby;
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+ 
+  meta = with lib; {
+    description = "Flexibly import bank account CSV files into Ledger for command line accounting";
+    license = licenses.mit;
+    maintainers = "mckean.kylej@gmail.com";
+    platforms = platforms.unix;
+  };
```

Comment:
There is an issue right now with bundlerEnv where other binaries are being exported as well. Check out the `cide` package to see how to only keep what you want.